bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Image.java

```diff
@@ -162,6 +169,26 @@ default String getTitle() {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * Returns the value for the image's {@code uuid} attribute, if one was set.
+     *
+     * @return the value for the image's {@code uuid} attribute, if one was set, or {@code null}
+     * @since com.adobe.cq.wcm.core.components.models 11.0.0; marked <code>default</code> in 12.4.0
+     */
+    default String getUuid() {
+        throw new UnsupportedOperationException();
+    }
+
+    /**
+     * Checks if UUID is disabled on the asset.
+     *
+     * @return {@code true} if UUID is disabled on the asset, {@code false} otherwise
+     * @since com.adobe.cq.wcm.core.components.models 12.4.0
+     */
+    default boolean isUuidDisabled() {
```

Comment:
What do we need this API for? Can't we just determine this from a null UUID?

Reply:
This is used to give user the option of disabling the uuid field from the design dialog.

Reply:
@Samyak1080 - yes we have that option in the design dialog, I'm just wondering if we really need to provide it as API in the Image model, it's not used and is an implementation detail.